### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by futuretap** [werthdavid/homebridge-weather#10]
The plugin (version 1.4.0) just crashed with this backtrace:

```
/usr/local/lib/node_modules/homebridge-weather/index.js:120
                    temperature = parseFloat(this.cachedWeatherObj["main"]["temp"]);
                                                                          ^

TypeError: Cannot read property 'temp' of undefined
    at WeatherAccessory.returnTemp (/usr/local/lib/node_modules/homebridge-weather/index.js:120:75)
    at WeatherAccessory.<anonymous> (/usr/local/lib/node_modules/homebridge-weather/index.js:78:48)
    at Request._callback (/usr/local/lib/node_modules/homebridge-weather/index.js:195:21)
    at Request.self.callback (/usr/local/lib/node_modules/homebridge-weather/node_modules/request/request.js:186:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/usr/local/lib/node_modules/homebridge-weather/node_modules/request/request.js:1163:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-weather/node_modules/request/request.js:1085:12)

```

**Comment by werthdavid** [comment]
Could you provide your according conifg?

**Comment by futuretap** [comment]
```
{
  "accessory": "Weather",
  "apikey": "<redacted>",
  "locationByCoordinates": "lat=48.0&lon=11.1",
  "name": "OpenWeatherMap Temperature",
  "pollingInterval": "15"
}
```

**Comment by werthdavid** [comment]
Should be fixed in 1.4.1
